dataset.ttl

```diff
@@ -32,10 +32,10 @@ data:AdavaleBasin
     time:hasTime ischart:Devonian ;
     sdo:name "Adavale Basin"@en .
 
-data:AnakieProvince
+data:AnakieProvince dct:hasPart data:FORK_LAGOONS_SUBPROVINCE , dct:isPartOf data:THOMSON_OROGEN , data:NEOPROTEROZOIC_LOWER_PALEOZOIC_PROVINCES ,
         a geo:Feature ,
         sdo:Landform ,
-        geof:Province ;
+        geof:TectonisedMetomorphosedProvince ;

```

Comment:
The syntax here is giving me validation errors and I think this format is repeated throughout the file on multiple objects.

I believe it needs to look more like the below, declaring what the object is e.g. feature and landform then providing the dct:hasPart and isPartOf and semi-colon separated declarations

data:AnakieProvince 
        a geo:Feature ,
        sdo:Landform ,
        geof:TectonisedMetomorphosedProvince ;
    dct:hasPart data:FORK_LAGOONS_SUBPROVINCE ;
    dct:isPartOf data:THOMSON_OROGEN , data:NEOPROTEROZOIC_LOWER_PALEOZOIC_PROVINCES ;
    geo:hasGeometry [ a geo:Geometry ;

Reply:
a typo: TectonisedMetomorphosedProvince
It's in there so many times, and I cannot believe that I have overlooked it before now!!

Semicolons inserted.